projects/canopy/src/lib/docs/principles/accessibility/developer.stories.mdx

```diff
@@ -0,0 +1,225 @@
+
+<Meta title="Principles/Accessibility/Developer" parameters={{ viewMode: 'docs', previewTabs: { canvas: { hidden: true } } }} />
+
+# For developers
+
+<p class="standfirst">Here we provide information on developing content to be accessible.</p>
+
+## Developing with accessibility in mind
+
+A great deal of web content can be made accessible just by making sure the correct HTML elements are used for the correct purpose at all times.
+
+WAI-ARIA, the Accessible Rich Internet Applications Suite, can be used to make web content and web applications more accessible to people with disabilities.
+
+WAI-ARIA is especially helpful when creating dynamic content and advanced user interface controls developed with Ajax, HTML, JavaScript, and related technologies.
+
+
+**Components from the Canopy Angular library have been built to be accessible.**
+
+### Defining the language of the document
+
+Defining the language used in an HTML page has many benefits. It explicitly tells the browser what language you are using, thus improving SEO. Third-party translation tools and browsers can use the language information to use the correct dictionary. Additionally, assistive technologies can choose the suitable voice profile or character set.
+
+```html
+<html lang="en">
+  ...
+</html>
+```
+
+If you switch language within a document you can use the lang attribute on specific tags:
+
+```html
+<p>You could have <i lang="it">una frase in italiano<i></p>
+```
+
+### Provide a "skip to main content" link
+
+When keyboard-only users interact with your site they use the tab key to jump from link to link. If you have a lot of links in your header or in a menu, they must tab through those every time they come to a new page just to get to the main content. Providing a skip to main content link allows them to easily bypass this.
+
+You can do that by using the class `skip-content`:
+
+```css
+.skip-content {
+  background: var(--color-white);
+  border: 0 !important;
+  clip: rect(0 0 0 0) !important;
+  height: auto !important;
+  margin: 0 !important;
+  overflow: hidden !important;
+  padding: 0 !important;
+  position: absolute !important;
+  width: 0.0625em !important;
+  white-space: nowrap !important;
+
+  &:focus,
+  &:active {
+    left: 0;
+    clip: auto !important;
+    width: auto !important;
+    height: auto !important;
+    z-index: var(--z-index-skip-link);
+  }
+}
+```
+
+and in your HTML:
+
+```html
+<a href="#id" class="skip-content">Skip to main content</a>
+  ...
+<main id="id">
+  ...
+```
+
+
+
+This is automatically added in Canopy by using the [page component](./?path=/docs/components-primary-message-guide--page).
+
+### Images and alt attributes
+
+If an image is used as content it will need the `alt` attribute to describe the images content. If the image is purely decorative or doesn't add any valuable information you can either embed it as a background-image through CSS or setting the `alt` attribute to an empty string:
```

Comment:
I think the first sentence might be misinterpreted 🤔 I have changed it to:

If an image is used as content it will need the `alt` attribute to describe its appearance or function. If the image is purely decorative or doesn't add any valuable information you can either embed it as a background-image through CSS or setting the `alt` attribute to an empty string: